timeseries/src/autogluon/timeseries/models/abstract/model_trial.py

```diff
@@ -26,17 +26,17 @@ def model_trial(
     """Runs a single trial of a hyperparameter tuning. Replaces
     `core.models.abstract.model_trial.model_trial` for timeseries models.
     """
-    try:
-        model = init_model(
-            args, model_cls, init_params, backend=hpo_executor.executor_type, is_bagged_model=is_bagged_model
-        )
-        model.set_contexts(path_context=os.path.join(model.path_root, model.name))
+    model = init_model(
+        args, model_cls, init_params, backend=hpo_executor.executor_type, is_bagged_model=is_bagged_model
+    )
+    model.set_contexts(path_context=os.path.join(model.path_root, model.name))
 
-        train_data = load_pkl.load(train_path)
-        val_data = load_pkl.load(val_path)
+    train_data = load_pkl.load(train_path)
+    val_data = load_pkl.load(val_path)
 
-        eval_metric = model.eval_metric
+    eval_metric = model.eval_metric
 
+    try:
```

Comment:
thanks, i think this is a nice cleanup anyway. So we can keep it. It makes sense to put only fit inside try except block.